Incident Wash-Up

```diff
@@ -0,0 +1,19 @@
+Incident Wash-Ups should be used to discuss issues that have affected our customers and how we can help to avoid similar situations from occurring in the future. The Wash-Up should include all relevant parties at their earliest opportunity after the event. 
+
+Please use this format for all Incident Wash-Ups so that we have a consistent set of data on how we handle situations. Please add all notes from Incident Wash-Ups to the Confluence space (https://holidayextras.jira.com/wiki/display/WEB/Incident+Wash-Ups), for reference.
```

Comment:
I think storing them in github takes away some of their accessibility - github is not a tool that everyone in our team is familiar with and we shouldn't be expecting everyone to be familiar with it (e.g. Product Owners, Specialists), but these notes should be accessible to everyone. That's why I thought Louis' suggestion of Confluence was a more team-friendly one.